pkgs/applications/version-management/cvs/default.nix

```diff
@@ -11,6 +11,7 @@ stdenv.mkDerivation {
   patches = [
     ./getcwd-chroot.patch
     ./CVE-2012-0804.patch
+    ./CVE-2017-12836.patch
```

Comment:
Could we just `fetchPatch` it?

Reply:
No, unfortunately all the version I found (debian, MirBSD, OpenBSD,…) diverge a lot from our savannah based version 😞

Reply:
Also (our) upstream is kinda dead... we might want to consider using the debian version or extracing CVS from one of the *BSD CVSs'

Reply:
Ah, OK, let's merge this for now then